source/ArrowKeyStepper/ArrowKeyStepper.js

```diff
@@ -14,11 +14,12 @@ export default class ArrowKeyStepper extends PureComponent {
   };
 
   static propTypes = {
-    children: PropTypes.func.isRequired,
+    children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]).isRequired,
     className: PropTypes.string,
     columnCount: PropTypes.number.isRequired,
     disabled: PropTypes.bool.isRequired,
     mode: PropTypes.oneOf(['cells', 'edges']),
+    onScrollToChange: PropTypes.func,
```

Comment:
Seems like it's at least feasible that someone _might_ want to observe scroll changes without being in controlled mode. I wonder if controlled mode should be a more explicit flag (eg a `isControlled` boolean prop or something)?